src/SHCore.Desktop/SHCore+SHELL_UI_COMPONENT.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Identifies the type of UI component that is needed in the shell
+    /// </summary>
+    public enum SHELL_UI_COMPONENT
```

Comment:
Good question. I guess all the types in this package are applicable to multiple libraries, right?
If we want to namespace them differently, instead of using nested types as we typically do in library packages, we should just use a namespace. That seems reasonable to me.